### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by erikagayan**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,6 +1,28 @@
+import os
+import requests
+
+
+CITY = "Paris"
+KEY = os.environ["API_KEY"]
+URL = f"https://api.weatherapi.com/v1/current.json?q={CITY}"
```
Comment (YuriiKindrat) [inline app/main.py]: it is better to use `params` param instead of f-string for urls params

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]
what about this point ?
image

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]
what about this point? 
`You must modify this line with correct command to pull your image: COMMAND=docker pull <YOUR_DOCKER_ID/YOUR_IMAGE_NAME>.`

**Review by LiudmylaKulinchenko — APPROVED** [review]
Good job! 😎🍬

**Pull request closed without merging**